tests/std/tests/P2474R2_views_repeat/test.cpp

```diff
@@ -0,0 +1,294 @@
+
+    assert(first < second);
+    assert(!(second < first));
+    static_assert(noexcept(first < second)); // strengthened
+
+    assert(!(first > second));
+    assert(second > first);
+    static_assert(noexcept(first > second)); // strengthened
+
+    assert(first <= second);
+    assert(!(second <= first));
+    static_assert(noexcept(first <= second)); // strengthened
+
+    assert(!(first >= second));
+    assert(second >= first);
+    static_assert(noexcept(first >= second)); // strengthened
+
+    assert(first <=> second < 0);
+    assert(second <=> first > 0);
+    static_assert(noexcept(first <=> second)); // strengthened
+
+    {
+        I i = first;
+
+        assert(&++i == &i);
+        assert(i == second);
+        static_assert(noexcept(++i)); // strengthened
+
+        i = first;
+
+        assert(i++ == first);
+        assert(i == second);
+        static_assert(noexcept(i++)); // strengthened
+
+        assert(&--i == &i);
+        assert(i == first);
+        static_assert(noexcept(--i)); // strengthened
+
+        i = second;
+
+        assert(i-- == second);
+        assert(i == first);
+        static_assert(noexcept(i--)); // strengthened
+
+        assert(i + 1 == second);
+        static_assert(noexcept(i + 1)); // strengthened
+
+        assert(1 + i == second);
+        static_assert(noexcept(1 + i)); // strengthened
+
+        assert(&(i += 1) == &i);
+        assert(i == second);
+        static_assert(noexcept(i += 1)); // strengthened
+        assert(i - 1 == first);
+        static_assert(noexcept(i - 1)); // strengthened
+
+        assert(&(i -= 1) == &i);
+        assert(i == first);
+        static_assert(noexcept(i -= 1)); // strengthened
+
+        assert(second - first == 1);
+        static_assert(noexcept(second - first)); // strengthened
+
+        assert(first[1] == *second);
+        assert(second[-1] == *first);
+        static_assert(noexcept(first[1])); // strengthened
+    }
+
+    const same_as<ranges::sentinel_t<R>> auto last = rng.end();
+    assert(!(first == last));
+    static_assert(noexcept(first == last)); // strengthened
+    assert(first != last);
+    static_assert(noexcept(first != last)); // strengthened
+    if constexpr (bounded) {
+        assert(cmp_equal(last - first, rng.size()));
+        static_assert(noexcept(last - first)); // strengthened
+    }
+}
+
+struct move_tester {
+    int x         = -1;
+    move_tester() = default;
+    constexpr move_tester(move_tester&& other) noexcept {
+        other.x = 1;
+    }
+};
+
+struct forward_tester {
+    int x            = -1;
+    forward_tester() = default;
+    constexpr forward_tester(forward_tester& other) : x(1) {
+        other.x = 1;
+    }
+    constexpr forward_tester(forward_tester&& other) : x(2) {
+        other.x = 2;
+    }
+};
+
+struct tuple_tester {
+    forward_tester y;
+    forward_tester z;
+
+    // clang doesn't support parenthesized initialization of aggregates
+    // TRANSITION ???

```

Comment:
Because this is a Standard feature, you can just say `// TRANSITION, P0960R3: clang doesn't meow meow` here. Then we can easily look up its feature tables.

Additionally, if you wanted, you could test `__cpp_aggregate_paren_init` to begin testing the desired codepath in addition to the workaround.